torchvision/prototype/utils/_internal.py

```diff
@@ -126,3 +144,70 @@ def __delattr__(self, item: Any) -> NoReturn:
 
     def __repr__(self) -> str:
         return make_repr(type(self).__name__, self.items())
+
+
+def kwonly_to_pos_or_kw(
+    *,
+    param_map: Optional[Dict[str, str]] = None,
+    warn: bool = False,
```

Comment:
Are we ever going to call this with warn=False?

Reply:
I don't know. Let's resolve https://github.com/pytorch/vision/pull/5001/files#r759167502 first. If we remove `param_map`, the case for removing this is stronger, since it could simplify the whole decorator.

Reply:
My stance here is that if we're introducing keyword-only parameters, we should deprecate the use of positional arguments, and thus warn.

Otherwise this whole decorator seems like a no-op?

Reply:
It won't be a no-op since the conversion of the parameters would still happen, but no warning would be emitted. I'm ok with either behavior.

Reply:
My understanding is that this might be useful if we don't manage to finalize the deprecation of positional arguments across the entire torchvision before the release. One way to use it would be to maintain the decorator but inhibit the warning. This will make the conversion of the parameters but won't inform users about the deprecation until we are ready. This is an alternative course of action to removing the mandatory keyword args from the method signature. I don't have a strong opinion over which way to follow.

Reply:
I've decided to remove it since it makes the implementation easier now that https://github.com/pytorch/vision/pull/5001#discussion_r759167502 is resolved.